packages/gasket-plugin-fastify/package.json

```
@@ -50,11 +51,11 @@
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-json": "^3.1.0",
     "eslint-plugin-unicorn": "^44.0.0",
-    "fastify": "^3.3.0",
+    "fastify": "^4.11.0",
     "jest": "^29.3.1"
   },
   "peerDependencies": {
-    "fastify": "^3.0.0"
+    "fastify": "^4.0.0"
```

We should keep this backwards compatible.
```suggestion
    "fastify": "^3 || ^4"
```
Unless Fastify 4 is going to require a new major Gasket release from us?

---

I'm not sure if it will require a major release, there are some updates that were made in the `@gasket/plugin-swagger` package related to Fastify 4 that wouldn't work in Fastify 3 - is this something we can go over when we zoom?